--- meandmyshadow.desktop.orig	2012-03-05 20:27:38.000000000 +0200
+++ meandmyshadow.desktop	2012-05-06 21:48:57.380443484 +0300
@@ -1,10 +1,12 @@
 [Desktop Entry]
-Version=0.3
+Version=1.0
 Type=Application
 Name=MeAndMyShadow
-GenericName=Me And My Shadow
-Comment=MeAndMyShadow is a puzzle/platformer game
-TryExec=meandmyshadow
+GenericName=Me and My Shadow
+GenericName[tr]=Ben ve Gölgem
+Comment=A puzzle platformer
+Comment[tr]=Bir bulmaca platform oyunu
 Exec=meandmyshadow
-Icon=meandmyshadow
-Categories=Game;LogicGame;Game;
+Terminal=false
+StartupNotify=false
+Categories=Application;Game;LogicGame;
