--- CMakeLists.txt.orig	2012-03-26 20:44:50.000000000 +0300
+++ CMakeLists.txt	2012-04-01 16:42:33.394105700 +0300
@@ -541,16 +541,12 @@
 
 # Installing on *nix
 IF(UNIX)
-  SET(GAME_INSTALL_DIR "/usr/share/${GAME_UNIXNAME}")
-  SET(GAME_FINAL_DIR "/usr/bin")
+  SET(GAME_INSTALL_DIR "${DESTDIR}/usr/share/${GAME_UNIXNAME}")
+  SET(GAME_FINAL_DIR "${DESTDIR}/usr/bin")
 
   INSTALL(CODE "MESSAGE(\"Installing ${GAME_NAME} ${GAME_VERSION}... Make sure you have root privileges.\")")
-  INSTALL(TARGETS ${GAME_UNIXNAME} RUNTIME DESTINATION ${GAME_INSTALL_DIR})
+  INSTALL(TARGETS ${GAME_UNIXNAME} RUNTIME DESTINATION ${GAME_FINAL_DIR})
   INSTALL(FILES license.txt readme.html bg.png icon.png logo_big.png logo.png surge.png DESTINATION ${GAME_INSTALL_DIR})
   INSTALL(DIRECTORY characters src objects sprites config images levels licenses musics quests samples screenshots themes languages fonts ttf DESTINATION ${GAME_INSTALL_DIR} PATTERN ".svn" EXCLUDE)
 
-  INSTALL(CODE "MESSAGE(\"Creating files at ${GAME_FINAL_DIR}...\")")
-  INSTALL(CODE "EXECUTE_PROCESS(COMMAND \"cmake\" \"-E\" \"copy\" \"${GAME_INSTALL_DIR}/${GAME_UNIXNAME}\" \"${GAME_FINAL_DIR}/${GAME_UNIXNAME}\")")
-
-  INSTALL(CODE "MESSAGE(\"Done! Please run ${GAME_UNIXNAME} to start ${GAME_NAME}.\")")
 ENDIF(UNIX)
