--- ./tanglet.pro	2010-01-12 01:25:54.000000000 +0200
+++ ./tanglet.pro2	2010-01-15 15:27:50.811765807 +0200
@@ -40,7 +40,7 @@
 	src/window.cpp \
 	src/word_tree.cpp
 
-TRANSLATIONS = translations/tanglet_en.ts translations/tanglet_fr.ts
+TRANSLATIONS = translations/tanglet_en.ts translations/tanglet_fr.ts translations/tanglet_tr.ts
 
 RESOURCES = icons/icons.qrc data/data.qrc translations/translations.qrc
 macx:ICON = icons/tanglet.icns
@@ -59,11 +59,23 @@
 	data_fr.files = data/fr/*
 	data_fr.path = $$PREFIX/share/tanglet/data/fr/
 
+        data_tr.files = data/tr/*
+        data_tr.path = $$PREFIX/share/tanglet/data/tr/
+
+        translations_en.files = translations/tanglet_en.qm
+        translations_en.path = $$PREFIX/share/tanglet/translations/
+
+        translations_fr.files = translations/tanglet_fr.qm
+        translations_fr.path = $$PREFIX/share/tanglet/translations/
+
+        translations_tr.files = translations/tanglet_tr.qm
+        translations_tr.path = $$PREFIX/share/tanglet/translations/
+
 	icon.files = icons/tanglet.png
 	icon.path = $$PREFIX/share/icons/hicolor/48x48/apps
 
 	desktop.files = icons/tanglet.desktop
 	desktop.path = $$PREFIX/share/applications/
 
-	INSTALLS += target icon desktop data_en data_fr
+	INSTALLS += target icon desktop data_en data_fr data_tr translations_en translations_fr translations_tr
 }
