Index: Http.cpp
===================================================================
--- Http.cpp.orig
+++ Http.cpp
@@ -173,8 +173,10 @@ bool Http::post( const char* uri, const
       }
     } catch ( Wobbly w ) {
       fprintf(stderr,"http_get wobbly: %s\n",w.what());
+      return false;
     }
   }
+  return true;
 }
 
 
Index: Overlay.cpp
===================================================================
--- Overlay.cpp.orig
+++ Overlay.cpp
@@ -403,12 +403,14 @@ public:
 			       y - m_listarea.tl.y - (m_itemh+m_vgap)*i );
       renderItems();
     }
+    return true;
   }
 
   bool onDrag( int dx, int dy )
   {
     m_velocity.x = dx;
     m_velocity.y = dy;
+    return true;
   }
 
   void doRowUp() { pan(0,-5); }
Index: Game.h
===================================================================
--- Game.h.orig
+++ Game.h
@@ -58,7 +58,7 @@ struct GameControl
   virtual ~GameControl() {}
   virtual bool save( const char *file=NULL ) =0;
   virtual bool send() =0;
-  virtual bool load( const char* file ) {};
+  virtual bool load( const char* file ) { return false; };
   virtual void gotoLevel( int l, bool replay=false ) =0;
   virtual void clickMode(int cm) =0;
   Levels& levels() { return *m_levels; }
Index: Levels.cpp
===================================================================
--- Levels.cpp.orig
+++ Levels.cpp
@@ -114,6 +114,7 @@ bool Levels::addPath( const char* path )
 
 bool Levels::addLevel( const string& file, int rank, int index )
 {
+  return
   addLevel( getCollection(MISC_COLLECTION), file, rank, index );
 }
 
@@ -248,6 +249,7 @@ int Levels::collectionFromLevel( int i,
       }
     }
   }
+  return 0;
 }
 
 std::string Levels::collectionName( int i, bool pretty )
Index: Scene.cpp
===================================================================
--- Scene.cpp.orig
+++ Scene.cpp
@@ -616,6 +616,7 @@ bool Scene::activateStroke( Stroke *s )
 {
   activate(s);
   m_recorder.activateStroke( m_strokes.indexOf(s) );
+  return true;
 }
 
 void Scene::getJointCandidates( Stroke* s, Path& pts )
Index: Ui.cpp
===================================================================
--- Ui.cpp.orig
+++ Ui.cpp
@@ -1087,6 +1087,7 @@ bool Dialog::close()
     //fprintf(stderr,"close dialog\n");    
     m_closeRequested = true;
   }
+  return true;
 }
 
 
Index: Worker.cpp
===================================================================
--- Worker.cpp.orig
+++ Worker.cpp
@@ -64,4 +64,5 @@ int WorkerBase::startThread(void* wbase)
   event.user.data1 = wbase;
   event.user.data2 = 0;
   SDL_PushEvent(&event);
+  return 1;
 }
