--- opengoo-0.1b.orig/levelselector.cpp	2012-03-13 11:32:12.000000000 +0200
+++ opengoo-0.1b/levelselector.cpp	2012-03-17 00:25:53.928715163 +0200
@@ -20,7 +20,7 @@
 void LevelSelector::findLevels()//Find .level file in the current direcotory
 {
     QDir *dir;
-    dir=new QDir();
+    dir=new QDir("/usr/share/opengoo");
     QStringList filters;
     filters <<"*.level";
     dir->setNameFilters(filters);
