diff -Naur wxWidgets-2.9.1.orig/configure wxWidgets-2.9.1/configure
--- wxWidgets-2.9.1.orig/configure	2010-07-18 16:39:20.000000000 +0300
+++ wxWidgets-2.9.1/configure	2011-04-09 12:26:01.726000001 +0300
@@ -26434,7 +26434,7 @@
 { echo "$as_me:$LINENO: result: $wx_cv_std_libpath" >&5
 echo "${ECHO_T}$wx_cv_std_libpath" >&6; }
 
-SEARCH_LIB="`echo "$SEARCH_INCLUDE" | sed s@include@$wx_cv_std_libpath@g` /usr/$wx_cv_std_libpath"
+SEARCH_LIB="`echo "$SEARCH_INCLUDE" | sed s@include@$wx_cv_std_libpath@g` /usr/$wx_cv_std_libpath /usr/lib"
 
 if test "$build" != "$host" -a "$GCC" = yes; then
                     if cross_root=`$CC -print-prog-name=ld 2>/dev/null`; then
