--- CMakeLists.txt.orig	2011-07-15 22:36:15.000000000 +0300
+++ CMakeLists.txt	2011-07-19 21:24:21.000000000 +0300
@@ -32,13 +32,13 @@
 set(Boost_DETAILED_FAILURE_MSG 1)
 # Boost versions that were not released when CMake 2.8 came out.
 # You might have to update this list if you use a newer version of Boost.
-set(Boost_ADDITIONAL_VERSIONS "1.45 1.45.0 1.46 1.46.0 1.46.1")
+set(Boost_ADDITIONAL_VERSIONS "1.44.0 1.45 1.45.0 1.46 1.46.0 1.46.1")
 # Note: Some older versions of Boost cause compilation errors if compiled with
 # GCC option -std=c++0x. It might be possible to compile Pentobi with an older
 # version of Boost depending on the version of GCC (e.g. the combination
 # GCC 4.4/Boost 1.40 in Ubuntu 10.04 worked but the combination
 # GCC 4.5/Boost 1.42 in Ubuntu 11.04 causes compilation errors.
-find_package(Boost 1.45 REQUIRED
+find_package(Boost 1.44.0 REQUIRED
   COMPONENTS thread system filesystem program_options)
 
 find_package(Threads)
