--- makefile.orig	2011-08-20 13:51:42.000000000 +0300
+++ makefile	2011-08-28 13:50:59.624000007 +0300
@@ -2,7 +2,7 @@
 # Type 'make' then 'make install' to complete the installation of the library
 
 # For 'make install' to work, set LOCAL according to your system configuration
-LOCAL = /usr/local
+LOCAL = /usr
 
 # If you want to build a shared library, then run make with SHARED=1 and VERSION=version
 ifdef SHARED
@@ -21,7 +21,7 @@
 	OBJDIR = obj-iphone
 	LIBDIR = ../../lib-iphone
 	CXX = $(IPHONEBIN)/arm-apple-darwin9-g++-4.0.1 
-	CXXFLAGS += -g -Wall -fPIC  -isysroot /Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS2.2.sdk
+	CXXFLAGS +=  -Wall -fPIC  -isysroot /Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS2.2.sdk
 else ifeq ($(TARGETPLATFORM), android)
 	ANDROIDNDKROOT = /cygdrive/c/android/android-ndk-1.6_r1
 	ANDROIDBIN = $(ANDROIDNDKROOT)/build/prebuilt/windows/arm-eabi-4.2.1/bin
@@ -29,7 +29,7 @@
 	OBJDIR = obj-android
 	LIBDIR = ../../lib-android
 	CXX = $(ANDROIDBIN)/arm-eabi-gcc
-	CXXFLAGS += -g -Wall -I$(SYSROOT)/usr/include \
+	CXXFLAGS +=  -Wall -I$(SYSROOT)/usr/include \
 	-Wall \
 	-DANDROID \
 	-fno-exceptions \
@@ -41,7 +41,7 @@
 	CXX ?= gcc
 	# On i686 architecture you may need to add -march=i686 if you get 
 	# an undefined symbol for __sync_sub_and_fetch_4 in as_atomic.cpp.
-	CXXFLAGS += -g -Wall -fPIC -fno-strict-aliasing
+	CXXFLAGS +=  -Wall -fPIC -fno-strict-aliasing
 endif
 
 AR ?= ar
@@ -128,19 +128,19 @@
 clean:
 	$(DELETER) $(OBJ) $(BIN) $(LIBDIR)/$(DEVLIB)
 
-install: $(BIN)
-	@echo Installing to: $(LOCAL)/lib and $(LOCAL)/include...
+install:
+	@echo Installing to: $(DESTDIR)$(LOCAL)/lib and $(DESTDIR)$(LOCAL)/include...
 	@echo -------------------------------------------------------------------
 ifdef SHARED
-	$(COPIER) $(LIBDIR)/$(DEVLIB) $(LOCAL)/lib
+	$(COPIER) $(LIBDIR)/$(DEVLIB) $(DESTDIR)$(LOCAL)/lib
 endif
-	$(COPIER) $(BIN) $(LOCAL)/lib
-	$(COPIER) $(INCDIR)/$(INC) $(LOCAL)/include
+	$(COPIER) $(BIN) $(DESTDIR)$(LOCAL)/lib
+	$(COPIER) $(INCDIR)/$(INC) $(DESTDIR)$(LOCAL)/include
 	@echo -------------------------------------------------------------------
 	@echo Angelscript library installed. Enjoy!
 
 uninstall:
-	$(DELETER) $(LOCAL)/include/$(INC) $(LOCAL)/lib/$(LIB) $(LOCAL)/lib/$(DEVLIB)
+	$(DELETER) $(DESTDIR)$(LOCAL)/include/$(INC) $(DESTDIR)$(LOCAL)/lib/$(LIB) $(DESTDIR)$(LOCAL)/lib/$(DEVLIB)
 	@echo -------------------------------------------------------------------
 	@echo Angelscript library uninstalled.
 
