diff -Naur barrage-1.0.3.orig/configure.in barrage-1.0.3/configure.in
--- barrage-1.0.3.orig/configure.in	2010-12-24 13:01:38.000000000 +0200
+++ barrage-1.0.3/configure.in	2011-03-27 14:25:05.450000015 +0200
@@ -33,7 +33,7 @@
 AC_SUBST(audio_flag)
 AC_SUBST(audio_lib)
 
-src_dir="$datadir/games/barrage"
+src_dir="$datadir/barrage"
 
 dnl check if installation was disabled
 AC_ARG_ENABLE( install,
diff -Naur barrage-1.0.3.orig/src/menu.h barrage-1.0.3/src/menu.h
--- barrage-1.0.3.orig/src/menu.h	2005-02-14 20:50:23.000000000 +0200
+++ barrage-1.0.3/src/menu.h	2011-03-27 14:25:47.515000017 +0200
@@ -16,7 +16,7 @@
 #define MENU_H
 
 typedef struct {
-	char 	name[20]; 	/* name */
+	char 	name[24]; 	/* name */
 	int  	x, w, h; 	/* size and x position; y depends on the position
 				   in the menu and is computed dynamically */
 	int	id;		/* unique id */
