--- build/linux/Makefile.old	2008-03-19 21:58:51.000000000 +0100
+++ build/linux/Makefile	2009-03-08 23:43:07.000000000 +0100
@@ -1,15 +1,16 @@
 EXE = roadfighter
 SRC = src
-PREFIX = /usr
+PREFIX = $(DESTDIR)/usr
 EXTRADIR = ./build/linux
 
-GAMEDIR = $(PREFIX)/games
+GAMEDIR = $(PREFIX)/share
 STARTUP = $(GAMEDIR)/$(EXE)
-BINDIR = $(PREFIX)/share/games/$(EXE)
+BINDIR = $(PREFIX)/bin
 ICNDIR = $(PREFIX)/share/pixmaps
 APPDIR = $(PREFIX)/share/applications
+DATADIR = $(PREFIX)/share/$(EXE)
 
-DATA = fonts graphics maps sound readme.txt
+DATA = fonts graphics maps sound
 ICON = roadfighter.png
 DESKTOP = roadfighter.desktop
 
@@ -37,14 +38,15 @@
 	$(SRC)/main.o
 
 CC = g++
-CFLAGS = -g3 -O3 `sdl-config --cflags` -I/usr/X11R6/include
-LDFLAGS = `sdl-config --libs` -L/usr/X11R6/lib/ -lSDL_image -lSDL_mixer -lSDL_sound -lSDL_ttf
+CFLAGS += `sdl-config --cflags`
+LIBS = `sdl-config --libs` -lSDL_image -lSDL_mixer -lSDL_sound -lSDL_ttf
+LDFLAGS += 
 RM = rm -f
 CP = cp -r
 MD = mkdir -p
 ECHO = echo
 CHMOD = chmod
-STRIP = strip
+STRIP = echo
 
 all: $(EXE)
 
@@ -52,7 +54,7 @@
 	$(CC) $(CFLAGS) -c $< -o $@
 
 $(EXE): $(OBJS)
-	$(CC) $(LDFLAGS) $^ -o $@
+	$(CC) $(LDFLAGS) $^ -o $@ $(LIBS)
 	@$(STRIP) $@
 	@$(ECHO) " o If there are no errors, the game compiled succesfully"
 
@@ -61,19 +63,12 @@
 	@$(RM) $(SRC)/*.o $(SRC)/sge/*.o $(SRC)/*.bak core $(EXE)
 
 install: all
-	@$(ECHO) " o Creating install directory $(BINDIR)"
-	@$(MD) "$(BINDIR)"
-	@$(ECHO) " o Installing game and data to $(BINDIR)"
-	@$(CP) "$(EXE)" $(DATA) "$(BINDIR)"
-	@$(ECHO) " o Creating startup script $(STARTUP)"
-	@$(MD) "$(GAMEDIR)"
-	@$(ECHO) "cd \"$(BINDIR)\"; ./$(EXE); cd -" >"$(STARTUP)"
-	@$(CHMOD) 755 "$(STARTUP)"
-	@$(ECHO) " o Creating application menu entry"
+	@$(ECHO) " o Creating install directories"
+	@$(MD) "$(DATADIR)"
+	@$(ECHO) " o Installing game and data to $(DATADIR)"
+	@$(CP) "$(EXE)" $(DATA) "$(DATADIR)"
 	@$(MD) "$(ICNDIR)"
 	@$(CP) "$(EXTRADIR)/$(ICON)" "$(ICNDIR)"
-	@$(MD) "$(APPDIR)"
-	@$(CP) "$(EXTRADIR)/$(DESKTOP)" "$(APPDIR)"
 	@$(ECHO) ""
 
 uninstall:
