diff -Naur Caelum-0.5.0.orig/SConstruct Caelum-0.5.0/SConstruct
--- Caelum-0.5.0.orig/SConstruct	2009-08-09 10:48:07.000000000 +0300
+++ Caelum-0.5.0/SConstruct	2011-04-09 03:18:11.471000041 +0300
@@ -212,8 +212,8 @@
 
     if oisDemoEnv:
         oisDemoEnv.Program('CaelumDemo', 'samples/src/CaelumDemo.cpp')
-    if ceguiDemoEnv:
-        ceguiDemoEnv.Program('CaelumLab', 'samples/src/CaelumLab.cpp')
+#    if ceguiDemoEnv:
+#        ceguiDemoEnv.Program('CaelumLab', 'samples/src/CaelumLab.cpp')
     # This is how you add an ogre scratch exe:
     #demoEnv.Program('CaelumUnitTest', 'samples/src/CaelumUnitTest.cpp')
 
