diff -Naur powder-70.1-src.orig/includes/luaconsole.h powder-70.1-src/includes/luaconsole.h
--- powder-70.1-src.orig/includes/luaconsole.h	2011-12-19 23:40:51.000000000 +0200
+++ powder-70.1-src/includes/luaconsole.h	2012-01-07 14:55:40.857000011 +0200
@@ -5,9 +5,9 @@
 #include <lauxlib.h>
 #include <lualib.h>
 #else
-#include <lua5.1/lua.h>
-#include <lua5.1/lauxlib.h>
-#include <lua5.1/lualib.h>
+#include <lua.h>
+#include <lauxlib.h>
+#include <lualib.h>
 #endif
 #include <defines.h>
 
diff -Naur powder-70.1-src.orig/Makefile powder-70.1-src/Makefile
--- powder-70.1-src.orig/Makefile	2011-12-19 23:40:51.000000000 +0200
+++ powder-70.1-src/Makefile	2012-01-07 14:55:12.608000012 +0200
@@ -4,9 +4,9 @@
 
 CFLAGS := -w -std=c99 -D_POSIX_C_SOURCE=200112L -DLUACONSOLE -DGRAVFFT -Iincludes/ -D_GNU_SOURCE
 OFLAGS := -O3 -ffast-math -ftree-vectorize -funsafe-math-optimizations
-LFLAGS := -lpthread -lSDL -lfftw3f -lm -lbz2 -lX11 -llua5.1 -lrt
+LFLAGS := -lpthread -lSDL -lfftw3f -lm -lbz2 -lX11 -llua -lrt
 LFLAGS_X := -lm -lbz2 -lSDLmain
-LFLAGS_WIN := -lmingw32 -lgnurx -lws2_32 -lSDLmain -lpthread -lSDL -lfftw3f -lm -lbz2 -llua5.1
+LFLAGS_WIN := -lmingw32 -lgnurx -lws2_32 -lSDLmain -lpthread -lSDL -lfftw3f -lm -lbz2 -llua
 MFLAGS_SSE3 := -march=native -DX86 -DX86_SSE3 -msse3
 MFLAGS_SSE2 := -march=native -DX86 -DX86_SSE2 -msse2
 MFLAGS_SSE := -march=native -DX86 -DX86_SSE
