--- gamelib/game.py.orig	2008-09-14 01:36:48.000000000 +0300
+++ gamelib/game.py	2011-06-18 14:11:00.168000023 +0300
@@ -4,6 +4,8 @@
 import pygame
 import random
 import math
+import os
+import shutil
 
 class level_data:
     width = 1
@@ -176,7 +178,15 @@
     return mygame
     
 def continue_game(mygame, level):
-    f = data.load('game.sav')
+    # f = data.load('game.sav')
+    savedir = os.environ['HOME']+"/.stringrolled/"
+    savefile = savedir + "game.sav"
+    try:
+        os.stat(savefile)
+    except:
+        os.makedirs(savedir)
+        shutil.copy2('/usr/share/stringrolled/data/game.sav', savefile)
+    f = data.load(savefile)
     save = cPickle.load(f)
     f.close()
     mygame.cat.x = save.cat_x
@@ -210,7 +220,15 @@
     save.ball_x = x + 24
     save.ball_y = y + 16
     save.character = mygame.character
-    f = data.load('game.sav', 'wb')
+    # f = data.load('game.sav', 'wb')
+    savedir = os.environ['HOME']+"/.stringrolled/"
+    savefile = savedir + "game.sav"
+    try:
+        os.stat(savefile)
+    except:
+        os.makedirs(savedir)
+        shutil.copy2('/usr/share/stringrolled/data/game.sav', savefile)
+    f = data.load(savefile, 'wb')
     cPickle.dump(save, f)
     f.close()
     
