--- makefile.orig	2010-02-15 17:33:06.000000000 +0200
+++ makefile	2010-09-03 22:25:50.154067547 +0300
@@ -8,8 +8,8 @@
 USEPAK = 1
 
 PREFIX=$(DESTDIR)/usr
-BINDIR = $(PREFIX)/games/
-DATADIR = $(PREFIX)/share/games/$(PROG)/
+BINDIR = $(PREFIX)/bin/
+DATADIR = $(PREFIX)/share/$(PROG)/
 DOCDIR = $(PREFIX)/share/doc/$(PROG)/
 ICONDIR = $(PREFIX)/share/icons/hicolor/
 DESKTOPDIR = $(PREFIX)/share/applications/
@@ -87,9 +87,9 @@
 	mkdir -p $(ICONDIR)64x64/apps
 	mkdir -p $(DESKTOPDIR)
 
-	install -o root -g games -m 755 $(PROG) $(BINDIR)$(PROG)
-	install -o root -g games -m 644 $(PAKNAME) $(DATADIR)$(PAKNAME)
-	install -o root -g games -m 644 $(DOCS) $(DOCDIR)
+	install -o root -g root -m 755 $(PROG) $(BINDIR)$(PROG)
+	install -o root -g root -m 644 $(PAKNAME) $(DATADIR)$(PAKNAME)
+	install -o root -g root -m 644 $(DOCS) $(DOCDIR)
 	cp $(ICONS)$(PROG).png $(ICONDIR)32x32/apps/
 	cp $(ICONS)$(PROG)-mini.png $(ICONDIR)16x16/apps/$(PROG).png
 	cp $(ICONS)$(PROG)-large.png $(ICONDIR)64x64/apps/$(PROG).png
