--- makefile.orig	2011-12-30 20:16:41.000000000 +0200
+++ makefile	2012-01-07 19:45:05.934000028 +0200
@@ -22,7 +22,7 @@
 CXX       = gcc
 
 PREFIX = $(DESTDIR)/usr
-BIN_DIR = $(PREFIX)/games/
+BIN_DIR = $(PREFIX)/bin/
 DOC_DIR = $(PREFIX)/share/doc/$(PROG)/
 ICON_DIR = $(PREFIX)/share/icons/hicolor/
 DESKTOP_DIR = $(PREFIX)/share/applications/
@@ -32,7 +32,7 @@
 ifeq ($(OS),Windows_NT)
 DATA_DIR =
 else
-DATA_DIR = $(PREFIX)/share/games/edgar/
+DATA_DIR = $(PREFIX)/share/edgar/
 endif
 

