--- barrage.desktop.orig	2011-07-28 18:00:11.000000000 +0300
+++ barrage.desktop	2011-08-02 12:40:47.317000004 +0300
@@ -1,7 +1,14 @@
 [Desktop Entry]
+Type=Application
+Version=1.0
+Encoding=UTF-8
 Name=Barrage
-Comment=A rather destructive action game.
+GenericName=Shooting Game
+GenericName[tr]=Atış Oyunu
+Comment=A rather destructive action game
+Comment[tr]=Yıkıcı bir aksiyon oyunu
 Exec=barrage
-Icon=barrage.png
-Type=Application
-Categories=Game;
+Icon=barrage
+Terminal=false
+StartupNotify=false
+Categories=Application;Game;ActionGame;
